corehq/apps/style/static/style/ko/components/inline_edit_v2.js

```diff
@@ -0,0 +1,182 @@
+/* global DOMPurify */
```

Comment:
Wondering if it's be worth moving this file into app manager temporarily, for the sake of taking advantage of the v1/v2 split and the diffs. Thoughts?